sdks/openai/simplified completion creator/05e5a443-51ba-4e60-9ce6-f923963c5c21.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "sdks/openai/Simplified completion creator",
+  "description": "Creates a completion for the provided prompt.\n\nIt uses text-davinci-003 as model.",
+  "interface": {
+    "inputs": {
+      "77151a8d-f339-4d97-ac15-e6cdd6a9a0cb": {
+        "type": "{\"prompt\": string, optional \"max_tokens\": (number or null)}",
+        "name": "request",
+        "description": "Receives the request for creating the completion.\n\nIt uses text-davinci-003 as model.\nThe max_tokens default is 1000.\n\nExample:\n```\n{\n  \"prompt\": \"What is the third planet?\"\n}\n```"
+      },
+      "8237fcf6-04af-4582-96ce-81d8c660f71e": {
+        "type": "{\"apiKey\": string}",
+        "name": "params",
+        "description": "Receives generic OpenAI params.\n\nExample: \n```\n{\"apiKey\": \"sk-XXXXXXXXXXYYYYYYYZZZZZZZnx\"}\n```"
+      }
+    },
+    "outputs": {
+      "b362d0b8-194a-4d53-8893-3b3d3f7c4b7f": {
+        "type": "string",
+        "name": "completion",
+        "description": "Sends the completion.\n\nExample:\n```\n\"\\n\\nThe third planet from the sun is Earth.\" \n```"
```

Comment:
Should we call it as "best completion" or "first completion" since there are multiple completions in the response and that way it is ambiguous. Or do not output response just the completion?